MAP/src/components/StatsBar/StatsBar.js

```diff
@@ -30,31 +35,33 @@ const StatsBar = ({
     }, [startDate, endDate, selectedDate, daysRange]);
 
     return (
-        <div className={styles.statsBar}>
+        <div className={`${isOpen ? null : styles.closed} ${styles.statsBar}`}>
             <div
                 className={styles.lockdownLogo}
             >
                 {logoSimple}
             </div>
-            <div className={styles.relativeWrapperOne}>
-                <p className={styles.territoriesInLockdown}>
-                    {t('header.totals.territoriesLockdown')}
-                </p>
-                <p className={styles.numLockdown}>
-                    {separateNumber(totalsData.lockdown, t('languageId'))}
-                </p>
-            </div>
-            <div className={styles.relativeWrapperTwo}>
-                <p className={styles.numAffected}>
-                    {separateNumber(totalsData.affected, t('languageId'))}
-                </p>
-                <p className={styles.territoriesInLockdown}>
-                    {t('header.totals.peopleAffected')}
-                </p>
-            </div>
-            <div className={styles.flexWrapperOne}>
+            {isOpen && <div className={styles.relativeWrapperOne}>
+                    <p className={styles.territoriesInLockdown}>
+                        {t('header.totals.territoriesLockdown')}
+                    </p>
+                    <p className={styles.numLockdown}>
+                        {separateNumber(totalsData.lockdown, t('languageId'))}
+                    </p>
+                </div>}
+            {isOpen && <div className={styles.relativeWrapperTwo}>
+                    <p className={styles.numAffected}>
+                        {separateNumber(totalsData.affected, t('languageId'))}
+                    </p>
+                    <p className={styles.territoriesInLockdown}>
+                        {t('header.totals.peopleAffected')}
+                    </p>
+                </div>}
+            <div
```

Comment:
let's create one Component and pass those props
1.styles.relativeWrapperOne
2.styles.territoriesInLockdown
3.totalsData.lockdown.